### ISSUE TITLE
"Exception: JNI: Init'd AndroidJavaClass with null ptr!" error when running in Unity Editor

### ISSUE THREAD

**Issue by wei780901** [adjust/unity_sdk#120]
```
Exception: JNI: Init'd AndroidJavaClass with null ptr!
UnityEngine.AndroidJavaClass..ctor (IntPtr jclass) (at C:/buildslave/unity/build/Runtime/Export/AndroidJavaImpl.cs:562)
UnityEngine.AndroidJavaObject.get_JavaLangClass () (at C:/buildslave/unity/build/Runtime/Export/AndroidJavaImpl.cs:540)
UnityEngine.AndroidJavaObject.FindClass (System.String name) (at C:/buildslave/unity/build/Runtime/Export/AndroidJavaImpl.cs:531)
UnityEngine.AndroidJavaClass._AndroidJavaClass (System.String className) (at C:/buildslave/unity/build/Runtime/Export/AndroidJavaImpl.cs:551)
UnityEngine.AndroidJavaClass..ctor (System.String className) (at C:/buildslave/unity/build/artifacts/generated/common/runtime/AndroidJavaBindings.gen.cs:92)
com.adjust.sdk.AdjustAndroid..cctor () (at Assets/Adjust/Android/AdjustAndroid.cs:16)
Rethrow as TypeInitializationException: An exception was thrown by the type initializer for com.adjust.sdk.AdjustAndroid
com.adjust.sdk.Adjust.start (com.adjust.sdk.AdjustConfig adjustConfig) (at Assets/Adjust/Adjust.cs:90)
com.adjust.sdk.Adjust.Awake () (at Assets/Adjust/Adjust.cs:43)
```

I use the sampleScene ExampleGUI

I have some issue...
I use sdk version 4.12.1 & unity version 5.6.2
I don't know to fix it.

**Comment by uerceg** [comment]
@wei780901 

Thanks for reporting, we're preparing SDK update that should solve these issues in Editor.

Will keep you posted.

**Comment by hdemmer** [comment]
For now we replace every `#if UNITY_IOS` in `Adjust.cs` with

    #if UNITY_EDITOR
    //  ... possible return here ...
    #elif UNITY_IOS

Beware that some of these functions return values so you might need to insert a `return string.Empty` or `return null` between these lines.

**Comment by uerceg** [comment]
@wei780901 

This should now be fixed with latest v4.12.2 version: https://github.com/adjust/unity_sdk/releases/tag/v4.12.2

@hdemmer 

Thanks for the suggestions, hopefully this should not be the issue anymore with latest version.

Feel free to reopen the issue if problem still persists with latest SDK version.

Cheers

**Comment by dev51312117411** [comment]
i encountered this error
help me with that error.
exception:JNI:Init'd AndroidJavaClass with null ptr!
UnityEngine.AndroidJavaClass..ctor(IntPtr jClass)
